packages/toggle-model/src/EnableByMatchingSegment.php

```diff
@@ -12,11 +12,23 @@
 final class EnableByMatchingSegment implements ToggleStrategy
 {
     public const NAME = 'enable_by_matching_segment';
+    private string $id;
     private Segments $segments;
 
-    public function __construct(Segments $segments)
+    public function __construct(string $id, Segments $segments)
     {
         $this->segments = $segments;
+        $this->id = $id;
```

Comment:
Here we could assign variables in the same order as parameters:
```suggestion
        $this->id = $id;
        $this->segments = $segments;
```